frontend/public/style/_common.scss

```diff
@@ -562,6 +563,9 @@
         color: $color-pf-blue-400;
       }
     }
+    &--disabled {
+      pointer-events: none;
```

Comment:
`.btn[disabled]` sets `cursor` to `not-allowed`.  Do we still want `pointer-events`?  I'm thinking no?  Sorry for missing this earlier, @sg00dwin (I blame my cold).